telethon/tl/custom/message.py

```diff
@@ -824,15 +826,15 @@ async def click(self, i=None, j=None,
                 This is equivalent to ``message.buttons[0][1].click()``.
 
             text (`str` | `callable`):
-                Clicks the first button with the text "text". This may
+                Clicks the first button or poll option with the text "text". This may
```

Comment:
Would it be desirable to have callable text or filter be allowed to be `True` for multiple poll options, or would that complicate its usage requiring a lot of extra care?

Reply:
In this case, we need to know whether select all or only one option. If many options are suitable then for a regular poll we should take the first suitable one, and for multiple choice - all of them. It`s not very clear. Maybe adding new argument `multiselect_options`, if callable returns `True` then select all suitable ones?

Reply:
Maybe like this a30ec32

Reply:
Polls already have a `multiple_choice` flag. Maybe we can do this: if it's `True`, the callables may match multiple options, and if not it will select the first matching option. This way we don't need yet another argument and people may use regex to match on multiple options by text, or something. My biggest fear was that it would accidentally select more than one option unintentionally and I wanted your opinion on this.

Reply:
I think that allowing multiple choice where it possible (`multiple_choice` is `True`) imposes additional complications as unwanted selections. And by default better to find first matched option or even disable multichoice with callables. 
Devs can do some preprocessing with polls by themselves and call `Message.click()` with array of desired options.
So in my opinion finding first one is more predictable behaviour. Or controling multichoice by using additional argument (`multiselect_options`).

Reply:
Let's stick with "callables are always select the first option" and remove the multiselect parameter. We could just document that those parameters can't be used for multiple choices and more things considered in the future if people finds a need for it.

Reply:
Yep, i added few commits about that.